src/Qowaiv.Validation.Abstractions/Result_TModel.cs

```diff
@@ -72,7 +66,7 @@ public Result<TOut> Act<TOut>(Func<TModel, Result<TOut>> action)
             if (IsValid)
             {
                 var outcome = action(Value);
-                return For(outcome.IsValid
+                return new Result<TOut>(outcome.IsValid
```

Comment:
Why do you need to specify a Result<TOut> here but not in the other methods?

Reply:
Updated!